### ISSUE TITLE
Change name of filter button to "Apply Filters"

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2524]
## Builds

- [staging](https://console.cloud.google.com/cloud-build/builds;region=global/3fa8885d-ddd1-4211-a9fe-70c246f1f4db?project=proudcity-1184)
- [master](https://console.cloud.google.com/cloud-build/builds;region=global/ec6c5989-87d9-4a72-bc02-8002cf336085?project=proudcity-1184)
  -  includes #2534 for login lock
  - includes #2537 to turn off XMLRPC

## Source

https://airtable.com/appRFUc45sF4yWFut/tbl51inE74gEWlB0x/viwG9G4vAjsjQEh4g/recuzE962cNqhepj3?blocks=hide

## Describe the bug

When the screen reader user heard "filter" he assumed it was a button to open a modal with more filters.  He bypassed it and expected that the filter he had checked would have been applied already. 	

## Recommendation

Recommend changing the name of the button to "Apply Filters" to make it clear that it should be selected in order for the selection to be aplied. 

## Screenshots

![filter](https://github.com/proudcity/wp-proudcity/assets/1569463/17bfc4e3-4c8a-465d-af47-4f22cc651841)

**Comment by curtismchale** [comment]
@kevindherman @lukefretwell this is deployed to beta. You can see the new button text here: https://beta.proudcity.com/events/

**Comment by kevindherman** [comment]
@curtismchale @lukefretwell looks good to me.

**Comment by curtismchale** [comment]
Deployed
Composer: https://github.com/proudcity/wp-proudcity/releases/tag/2024.04.04.0737
Build: https://github.com/proudcity/proud-recipes/releases/tag/2024.04.04.0737
